src/components/ListItem/ListItem.stories.jsx

```diff
@@ -13,25 +13,43 @@ storiesOf('Components|ListItem', module)
         notes: {markdown: markdownNotes}
     })
     .addDecorator(withKnobs)
-    .add('default', () => (
+    .add('Default', () => (
         <div style={{transform: 'scale(1)', height: '100vh'}}>
```

Comment:
Just to be more semantic, can we change these `div`s to an `ul` ?

Reply:
Done!